pkg/statdb/proto/statdb.proto

```diff
@@ -82,18 +83,36 @@ message FindValidNodesResponse {
 // UpdateRequest is a request message for the Update rpc call
 message UpdateRequest {
   Node node = 1;
-  bytes APIKey = 2;
 }
 
 // UpdateRequest is a response message for the Update rpc call
 message UpdateResponse {
   NodeStats stats = 1;
 }
 
+// UpdateUptimeRequest is a request message for the UpdateUptime rpc call
+message UpdateUptimeRequest {
```

Comment:
so the requests don't require an api key in the struct because that's being passed somewhere as a flag?